tcp_ip:
  sub_port: "5555"
  pub_port: "5555"
  req_port: "4444"
  rep_port: "4444"
  zipcode: "00000"

udp_ip:
  port: "8080"
  send_group: "FV1"
  recv_group: "LV"

socket:
  rad_flag: true
  dsh_flag: true
  req_flag: true
  rep_flag: false
  sub_flag: true
  pub_flag: false

params:
  target_vel: 0.00
  safety_vel: 0.45
  fv_max_vel: 1.20
  lv_stop_dist: 0.5
  fv_stop_dist: 0.3
  safety_dist: 1.5
  target_dist: 0.8
  angle_degree: 0.0
  Kp_d: 2.0
  Ki_d: 0.2
  K: 0.15
  a:
    a: 0.
    b: -0.37169
    c: 1.2602
    d: -1.5161
    e: 0.70696
  b:
    a: 0.
    b: -1.7536
    c: 5.0931
    d: -4.9047
    e: 1.6722

Calibration:
  matrix:
    a: 329.10805843346054
    b: 0.
    c: 320. 
    d: 0. 
    e: 329.10805843346054
    f: 240. 
    g: 0. 
    h: 0. 
    i: 1.
  dist_coef:
    a: -0.34335984453339752
    b: 0.14676511736197059
    c: 0.
    d: 0. 
    e: -0.032260115806245951

ROI:
  width: 640
  height: 480
  top_gap: 0.416
  bot_gap: 0.185
  top_height: 0.99
  bot_height: 0.47
  extra_f: 0.00
  extra_b: 0.00
  extra_up: 140
  extra_down: 180

